### PR TITLE
Search -> Map Integration

### PR DESCRIPTION
**Pull request by jarrencampos**
[TRELLO BOARD CARD](https://trello.com/c/AgZ7DkjQ/33-as-a-user-i-want-to-be-able-to-view-the-location-i-searched-for-on-a-map-on-my-mobile-app)

Location:

- MapScreenViewController.swift
- SearchViewController.swift
- +Map.swift

Details:

- Added map struct file
- Used search bar delagate to notify the app when the user hits entry and uses MKLocalSearch Requests to run a natural language query from the users input.
- uses a closure to confirm that the user input is a valid input.
- Once passed through error check, it passed the data to the MapScreenViewController file.
- Fixed the code to initialize the city label to accept the user input -> City

### CODE REVIEW

File: labs-ios-starter/Model/Alert.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//  Alert.swift
+//  labs-ios-starter
+//
+//  Created by Jarren Campos on 2/10/21.
+//  Copyright © 2021 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+import Foundation
+
```
Comment (FabiolaSaga) [inline labs-ios-starter/Model/Alert.swift:11]: It would be helpful to add documentation on here!

File: labs-ios-starter/Model/Map.swift [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+//  labs-ios-starter
+//
+//  Created by Jarren Campos on 2/10/21.
+//  Copyright © 2021 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import MapKit
+
```
Comment (FabiolaSaga) [inline labs-ios-starter/Model/Map.swift:12]: Same here, documentation

**Review by FabiolaSaga — COMMENTED** [review]
Looks good! added a couple of comments about adding documentation.

File: labs-ios-starter/Model/Map.swift (hunk 2 above)
Comment (jarrencampos) [reply labs-ios-starter/Model/Map.swift:12]: Added documentation for clarification. Thanks for the suggestion!

**Pull request merged**